packages/wp-source/src/libraries/handlers/__tests__/__snapshots__/post-type-with-query.tests.ts.snap

```diff
@@ -21,7 +21,16 @@ Object {
     },
   },
   "authorBase": "",
-  "category": Object {},
+  "category": Object {
+    "1": Object {
+      "count": 5,
+      "id": 1,
+      "link": "/category/cat-1/",
+      "parent": 0,
+      "slug": "cat-1",
+      "taxonomy": "category",
+    },
+  },
```

Comment:
In this case, the snapshot was wrong :sweat_smile:. The bugfix now makes the category with id `1` to be populated.